sampo/structurator/prepare_wg_copy.py

```diff
@@ -0,0 +1,83 @@
+from copy import deepcopy
+
+from sampo.schemas.graph import GraphNode, WorkGraph
+from sampo.schemas.utils import uuid_str
+from sampo.schemas.works import WorkUnit
+
+
+def copy_graph_node(node: GraphNode, new_id: int | str | None = None,
+                    change_id: bool = True) -> tuple[GraphNode, tuple[str, str]]:
+    """
+    Makes a deep copy of GraphNode without edges, with the id changed to a new randomly generated or specified one
```

Comment:
You state that this method **must** change id, but receive the parameter that can change this behaviour.